examples/helloworld.rs

```diff
@@ -0,0 +1,91 @@
+use std::sync::Arc;
+use tokio::sync::RwLock;
+
+use serde::{Serialize, Deserialize};
+use tokio::{signal::unix::{signal, SignalKind}, select};
+use valensas_vault::service::{VaultService, TokenRenewable};
+
+#[derive(Serialize, Deserialize, Clone)]
+struct MySpecialToken {
+    special_data: String 
+}
+
+#[tokio::main]
+async fn main() {
+    dotenv::dotenv().ok();
```

Comment:
Do we really need a dotenv here? There isn't a `.env` file in the repo anyway. I think we can just use `std::env:set_var` here instead as an example.